src/plinder/core/system/system.py

```diff
@@ -81,31 +103,79 @@ def archive(self) -> Path | None:
 
     @property
     def system_cif(self) -> str:
+        """
+        Path to the system.cif file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "system.cif").as_posix()
 
     @property
     def receptor_cif(self) -> str:
+        """
+        Path to the receptor.cif file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "receptor.cif").as_posix()
 
     @property
     def receptor_pdb(self) -> str:
+        """
+        Path to the receptor.pdb file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "receptor.pdb").as_posix()
 
     @property
     def system_pdb(self) -> str:
+        """
+        Path to the system.pdb file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "system.pdb").as_posix()
 
     @property
     def sequences(self) -> str:
+        """
+        Path to the sequences.fasta file
+
+        Returns
+        -------
+        str
+            path
+        """
         assert self.archive is not None
         return (self.archive / "sequences.fasta").as_posix()
 
     @property
     def chain_mapping(self) -> dict[str, Any] | None:
```

Comment:
Just a note that `chain_mapping` and `water_mapping` are only required for receptor.pdb, and receptor.pdb is not usable without `chain_mapping` so maybe would put them into the same function?

Reply:
These are merely "helpers" to get the files. If we want to materialize the receptor.pdb along with chain mapping and (optionally) water mapping, we'd do it in a separate function that would compose these together.